Framework/API/src/FileBackedExperimentInfo.cpp

```diff
@@ -3,8 +3,8 @@
 
 #include <sstream>
 
-#include <nexus/NeXusException.hpp>
 #include <nexus/NeXusFile.hpp>
+#include <nexus/NeXusException.hpp>
```

Comment:
Was there a reason to break the alphabetic ordering here? If this is a mistake, please have a look at `LoadMcStasNexus.cpp`, ``.